src/main/java/org/myrobotlab/service/meta/abstracts/Meta.java

```diff
@@ -0,0 +1,20 @@
+package org.myrobotlab.service.meta.abstracts;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.myrobotlab.framework.ServiceReservation;
+
+abstract public class Meta {
```

Comment:
Why this class and the MetaData class?  that seems silly to have both.. and the still kicking around ServiceType class.

Reply:
don't get me started.